src/transformers/models/gemma/modeling_gemma.py

```diff
@@ -55,25 +50,30 @@
     from flash_attn.bert_padding import index_first_axis, pad_input, unpad_input  # noqa
 
 
-# This makes `_prepare_4d_causal_attention_mask` a leaf function in the FX graph.
-# It means that the function will not be traced through and simply appear as a node in the graph.
-if is_torch_fx_available():
-    if not is_torch_greater_or_equal_than_1_13:
-        import torch.fx
+if is_flash_attn_2_available():
+    from flash_attn import flash_attn_func, flash_attn_varlen_func
+    from flash_attn.bert_padding import index_first_axis, pad_input, unpad_input  # noqa
+"""PyTorch Gemma model."""
```

Comment:
this is potentially weird, I gotta remove this duplicate ruff is not able to

Reply:
yes indeed, and sort imports back up as well (maybe ruff does that?)